### PR TITLE
Ubuntu Bionic

### PR DESCRIPTION
**Pull request by enricostano**

### CODE REVIEW

**Comment by markets** [comment]
@enricostano I think the latest commit can be reverted if we finally merge https://github.com/coopdevs/timeoverflow/pull/503.

File: .devenv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 NAME="timeoverflow"
 DISTRIBUTION="ubuntu"
-RELEASE="xenial"
+RELEASE="bionic"
 ARCH="amd64"
 HOST="local.$NAME.org"
 PROJECT_NAME="timeoverflow"
```
Comment (enricostano) [inline .devenv:6]: `PYTHON_INTERPRETER="python2.7"`

Can we use Python 3, now?

Comment (danypr92) [reply .devenv:6]: Yes, we can start to use the interpreter of Python 3.

Remember that to use the Python 3 we need to change the `ansible_python_interpreter` variable: 
https://github.com/coopdevs/timeoverflow-provisioning/blob/master/inventory/group_vars/all.yml#L2

File: .devenv [hunk 2]
```diff
@@ -1,6 +1,6 @@
 NAME="timeoverflow"
 DISTRIBUTION="ubuntu"
-RELEASE="xenial"
+RELEASE="bionic"
 ARCH="amd64"
 HOST="local.$NAME.org"
```
Comment (raneq) [inline .devenv]: We changed our default naming to
```suggestion
HOST="$NAME.local"
```
Is it ok to you to change it too?

**Review by raneq — CHANGES_REQUESTED** [review]
With regard of the other changes, I can't say more. And about python3, @danypr92 has confirmed that we can use python3 but requests some changes. I'll give my approval then :)

**Review by sauloperez — APPROVED** [review]

**Comment by sauloperez** [comment]
@raneq we finally decided to merge this and provide Python 3 support in a separate purpose-specific PR. I hope you don't mind.

**Comment by sauloperez** [comment]
As promised @raneq https://github.com/coopdevs/timeoverflow-provisioning/pull/71

**Pull request merged**